src/App.module.css

```diff
@@ -0,0 +1,55 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@700&display=swap');
+
+.mainContainer {
+    height: 100vh;
+    width: 100vw;
+    background-color: #1E213F;
+    color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+}
+
+.pomodoroText {
+    margin-top: 2rem;
+    margin-bottom: 0;
+    color: #D7E0FF;
+    text-align: center;
+    font-family: 'Kumbh Sans', sans-serif;
+    font-size: 1.5rem;
+    font-weight: 700;
+    line-height: normal;
+}
+
+@media (min-width: 30rem) {
+    .pomodoroText {
+        font-size: 2rem;
+    }
+}
+
+@media (min-width: 30rem) and (orientation: portrait) {
+    .pomodoroText {
+        margin-top: 5rem;
+    }
+}
+
+@media (min-width: 30rem) and (orientation: landscape) {
+    .pomodoroText {
+        margin-top: 5vh;
+    }
+}
+
+.settingsIcon {
+    margin-top: 4rem;
+    position: absolute;
+    bottom: 7%;
+}
+
+@media (min-width: 30rem) and (max-height: 46.875rem) and (orientation: landscape) {
```

Comment:
because on my monitor the height is 50rem and everything looks good, but on laptop it is 37.5rem and elements vere places on each other. 46.875rem was the moment, when it started to look good, so for smaller heights I decreased the width and height of elements. is it the right approach? how could I do it better?

Reply:
In general I would focus just on just screen width and made sure that for each possible width, elements don't cause appearance of horizontal scrollbar. If in some configuration content of the page becomes scrollable vertically, I wouldn't worry about it. However, in case of three conditions in media query and one with very specific number it may not be clear for another dev jumping into the project why it is implemented like that.

Reply:
also according to: https://polypane.app/blog/the-complete-guide-to-css-media-queries/ it's recommended to use em instead of rem to define breakpoints: "All the width and height media features support the regular CSS units like pixels and ems. It's recommended to use the em unit for media queries, so it scales nicely when people zoom in their browser. This won't happen with pixels, and rem is the same size as an em when it's used in media queries. rem also has some bugs in Safari. So, em's are best.

If you think sizing in ems is hard because you don't know how large an em is: media queries are handled at the top level, so 1 em is always 16 pixels or what the browser set as the default font size (and so are rems)."